src/lib/util/Config.ts

```diff
@@ -215,31 +183,36 @@ export class Config {
 	}
 
 	protected getBooleanConfigValue(propertyName: string, engine: ENGINE): Promise<boolean> {
-		return this.getConfigValue<boolean>(propertyName, engine, this.typeChecker.booleanCheck);
+		return this.getConfigValue<boolean>(propertyName, engine, booleanTypeGuard, 'InvalidBooleanValue');
 	}
 
 	protected getNumberConfigValue(propertyName: string, engine: ENGINE): Promise<number> {
-		return this.getConfigValue<number>(propertyName, engine, this.typeChecker.numberCheck);
+		return this.getConfigValue<number>(propertyName, engine, numberTypeGuard, 'InvalidNumberValue');
 	}
 
 	protected getStringArrayConfigValue(propertyName: string, engine: ENGINE): Promise<string[]> {
-		return this.getConfigValue<string[]>(propertyName, engine, this.typeChecker.stringArrayCheck);
+		return this.getConfigValue<string[]>(propertyName, engine, stringArrayTypeGuard, 'InvalidStringArrayValue');
```

Comment:
By removing the message in `Config.js`, we make it possible to just use a single message template in this method call.